website/src/docs/react-dashboard.md

```diff
@@ -40,6 +41,8 @@ class MusicUploadButton extends React.Component {
   }
 
   render () {
+    this.props.uppy.use(Webcam, { id: "Webcam" });
+
```

Comment:
Webcam will get `id: "Webcam"` by default, so I’m wondering if we should omit this here for simplicity.

Reply:
+1